document-store/src/main/java/org/hypertrace/core/documentstore/postgres/PostgresCollection.java

```diff
@@ -187,20 +220,21 @@ public boolean updateSubDoc(Key key, String subDocPath, Document subDocument) {
   }
 
   @VisibleForTesting
-  protected String parseFilter(Filter filter, Params.Builder paramsBuilder) {
+  protected String parseFilter(Filter filter, Params.Builder paramsBuilder, String aliasPrefix) {
```

Comment:
Can you please add the `@Nullable` notation for `aliasPrefix`?

Reply:
Done